makefiles/arch/cortexm.inc.mk

```
@@ -43,6 +43,32 @@ export USEMODULE += newlib
 # set default for CPU_MODEL
 export CPU_MODEL ?= $(CPU)
 
+
+# extract version inside the first parentheses
+ARM_GCC_VERSION = $(shell $(TARGET_ARCH)-gcc --version | sed -n '1 s/[^(]*(\([^\)]*\)).*/\1/p')
+
+# Ubuntu bionic gcc-arm-none-eabi compiler is not supported
+# Both when using gnu and llvm toolchains
+#
+#     /usr/bin/arm-none-eabi-gcc --version | head -n 1
+#     arm-none-eabi-gcc (15:6.3.1+svn253039-1build1) 6.3.1 20170620
+#
+ARM_GCC_UNSUPPORTED += 15:6.3.1%  # ubuntu bionic, ignore 'svn' build part
+
+# Should not raise an error on the host system version when building in docker
+ifneq (1,$(BUILD_IN_DOCKER))
+ifneq (,$(filter $(ARM_GCC_UNSUPPORTED),$(ARM_GCC_VERSION)))
```

Maybe use indentation here to make it more readable:

```suggestion
  ifneq (,$(filter $(ARM_GCC_UNSUPPORTED),$(ARM_GCC_VERSION)))
```

---

Sure, I update all lines.

---

Was addressed.